containers/docker/pwpush/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+export RAILS_ENV=production
+
+
+if [ -n "$PWP_DB" ]
+then
+    echo "Setting $PWP_DB database backend"
+    if [ "$PWP_DB" == "sqlite" ]
+    then
+        export DATABASE_URL=sqlite3:db/db.sqlite3
+    elif [[ "$PWP_DB" == "mysql" || "$PWP_DB" == "mariadb" ]]
+    then
+        export DATABASE_URL=mysql2://passwordpusher_user:passwordpusher_passwd@mysql:3306/passwordpusher_db
+    elif [ "$PWP_DB" == "postgres" ]
+    then
+        export DATABASE_URL=postgres://passwordpusher_user:passwordpusher_passwd@postgres:5432/passwordpusher_db
+    else 
+        echo "Unknown database backend $PWP_DB. Exiting..."
+        exit
+    fi
+    
+else 
+    echo "No database backend set. Exiting..."
+    exit
+fi
```

Comment:
Another approach might be that we drop `PWP_DB` entirely and rely solely on `DATABASE_URL`.  If `DATABASE_URL` isn't set, then we assume ephemeral/sqlite3 and output a container message saying so.

```sh
if [ -z "$DATABASE_URL" ]
    echo "DATABASE_URL not specified.  Assuming ephemeral backend.  Database may be lost on container restart."
    export DATABASE_URL=sqlite3:db/db.sqlite3
fi
```
I use "may be lost" above because some users back the `db/db.sqlite3` file with a docker mount.

For other existing users, this _shouldn't_  cause any issues - just change the container image.  For new users, it maintains simplicity: set `DATABASE_URL` if you want postgres/mariadb/mysql.

Reply:
I understand what you want to archieve and generally agree with it.

The single problem I can imagine is someone's using default values (for username/password/port and so on) by not setting `DATABASE_URL` variable at all.
In this case they would fall back to sqlite. But I suppose this would be acceptable

I'll implement it as soon as I'm back from a business trip in a couple of days

Reply:
implemented within latest commits